src/interactions/autocompleters/TrackerAutocomple.ts

```diff
@@ -0,0 +1,49 @@
+import { Autocompleter } from "../interactionClasses";
+import { AutocompleteInteraction } from "discord.js";
+import { trackerGames } from "../../db";
+import { blacklist, user } from "../../util/tracker/autocompleteHelper";
+
+class Tracker extends Autocompleter {
+	constructor() {
+		super("tracker");
+	}
+
+	async execute(interaction: AutocompleteInteraction): Promise<void> {
+		const sub = interaction.options.getSubcommand();
+		const focus = interaction.options.getFocused().toLowerCase() as string;
+		const option = interaction.options.getFocused(true).name;
+
+		// if user-option is given, use it, else default to executer
+		const userId =
+			typeof interaction.options.get("user")?.value == "string"
+				? interaction.options.get("user")?.value
+				: interaction.user.id;
```

Comment:
I think this can be written better. Are you testing if its a string in case its null? Just use ?? if yes